examples/devsite-demo/script.js

```diff
@@ -71,10 +71,9 @@ function init() {
   toolboxJson['contents'][2].name = getMsg('Math');
   toolboxJson['contents'][3].name = getMsg('Text');
   toolboxJson['contents'][4].name = getMsg('Lists');
-  toolboxJson['contents'][5].name = getMsg('Colour');
   // Separator.
-  toolboxJson['contents'][7].name = getMsg('Variables');
```

Comment:
@maribethb do you know why we're doing this instead of using Blockly's normal localization?

Reply:
Toolboxes don't really have support for dynamic messages afaik. I could see a couple different ways of solving this but I'm not sure which method you're referring to as "normal."

If you want history though, you might have to check with Neil. This is the old demo that's been on devsite for forever and I'm not sure where it came from before we moved it to samples.

Reply:
TIL, I thought you had to do this manually. I thought one of the old demos was doing something but I can't find it.